store/metadata.py

```diff
@@ -0,0 +1,6 @@
+class Metadata:
```

Comment:
Do we really need a separate class for this? We could just keep everything in `DataStore`.

Reply:
On second thoughts, let it be. But no need to create a separate file for it maybe?

Reply:
We could keep this class in store/base.py.